app/views/homescreen/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<%#-- copyright
+OpenProject is a project management system.
+Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See doc/COPYRIGHT.rdoc for more details.
+
+++#%>
+<% breadcrumb_paths(nil) %>
+<div class="homescreen-header">
+  <h2>
+    <span class="homescreen-header--icon"><%= organization_icon %></span>
+    <%= organization_name %>
+  </h2>
+</div>
+
+<section class="homescreen-blocks">
+<% homescreen_blocks.each do |block| %>
+  <% if block[:if].nil? || block[:if].call %>
+    <div class="homescreen-blocks--item">
+    <%= render partial: "homescreen/blocks/#{block[:partial]}", locals: (block[:locals] || {}) %>
+    </div>
+  <% end %>
+<% end %>
+</section>
+
+<section class="homescreen-links">
+<% homescreen_links.each do |link| %>
+  <% title = I18n.t(link[:label], scope: 'homescreen.links') %>
+  <a class="homescreen-link--item" href="<%= link[:url] %>" title="<%= title %>">
+    <span class="<%= link[:icon] %>"></span>
+    <%= title %>
+  </a>
+<% end %>
+</section>
+<% unless homescreen_socialmedia.empty? %>
+  <section class="homescreen-social-links">
+  <% homescreen_socialmedia.each do |link| %>
+    <a href="<%= link[:url] %>" title="<%= link[:name] %>" class="<%= link[:link_class] %>"></a>
```

Comment:
In the spirit of my "generalize social media" comment:

I'd also add a `link[:caption]` that will be displayed as text below the Icon (I assume that `link_class` will add fancy icons).
We can leave it empty for our own social media purposes and it won't harm anyone ^^